Source/WebKit/Shared/API/Cocoa/RemoteObjectInvocation.h

```diff
@@ -40,7 +40,7 @@ class RemoteObjectInvocation {
     WTF_MAKE_TZONE_ALLOCATED(RemoteObjectInvocation);
 public:
     struct ReplyInfo {
-        WTF_MAKE_TZONE_ALLOCATED(ReplyInfo);
+        WTF_MAKE_STRUCT_TZONE_ALLOCATED(ReplyInfo);
     public:
```

Comment:
could remove this `public:` while here

Reply:
Sure, I'll remove it since it isn't needed.